index.html

```diff
@@ -17,24 +17,24 @@ <h1>Біографія</h1>
     </ul>
```

Comment:
@volhets you need to add .css file to "marks.html" and for "gallery.html"
and provide styles the same